changes/13071-disable-fleetd-enroll-errors

```diff
@@ -0,0 +1,2 @@
+- EXPERIMENTAL: Applying the environmental variable SILENCE_FLEETD_ENROLL_ERROR=1 will silence fleetd errors if not setting `--fleet-url` when generating an orbit installer.
```

Comment:
```suggestion
- EXPERIMENTAL: Applying the environmental variable FLEETD_SILENCE_ENROLL_ERROR=1 will silence fleetd errors if not setting `--fleet-url` when generating an orbit installer.
```

should we document this somewhere or is intended to be a "hidden" feature for now?

Reply:
@zhumo thoughts?

Reply:
please don't block the merge on this, just a pass-by comment (still good idea to fix the typo in the release notes)

Reply:
Let's block the merge as the env var is wrong in the changelog :)

Reply:
🤦 i think i saw 3 different var names floating around

Reply:
Let's wait for product team to decide if we add the changelog or not. Also, this should be in `orbit/changes/`
(We haven't really used the orbit changelogs recently when releasing Orbit... we should fix this soon.)

Reply:
Definitely should be in the changelog. We should also document the existence of this in the public docs but make sure it is known that it is experimental and could be taken away anytime. See the original issue for more details on that.

Reply:
@zhumo Gotcha. What about the variable name? There's `SILENCE_FLEETD_ENROLL_ERROR` and `FLEETD_SILENCE_ENROLL_ERROR`.